crates/cairo-lint-core/src/lints/manual_unwrap_or_default.rs

```diff
@@ -0,0 +1,104 @@
+use cairo_lang_semantic::db::SemanticGroup;
+use cairo_lang_defs::plugin::PluginDiagnostic;
+use cairo_lang_diagnostics::Severity;
+use cairo_lang_semantic::{Expr, ExprIf, ExprMatch, Arenas};
+
+pub const MANUAL_UNWRAP_OR_DEFAULT: &str = "This can be done in one call with `.unwrap_or_default()`";
+
+/// Enum to represent an `if let` or `match` expression.
+pub enum IfLetOrMatch {
+    If(ExprIf),
+    Match (ExprMatch),
```

Comment:
is this copy pasted from the compiler ?

Reply:
Yeah to limit to limit the expr to just if else and match. 

https://github.com/starkware-libs/cairo/blob/main/crates/cairo-lang-semantic/src/expr/objects.rs#L136

Reply:
it doesn't work like that lol you need to use the already defined types you don't need to redefine anything

Reply:
Okay just realized how redundant it was since I was basically still matching them.

Alright will update them.